app/models/customer_order.rb

```diff
@@ -0,0 +1,12 @@
+class CustomerOrder < ApplicationRecord
+  belongs_to :user
+  belongs_to :product
+  belongs_to :lunch_order
+
+  validates :user_id, uniqueness: { scope: :lunch_order_id }
+  
+  def update_attribute(name, value)
+    send(name.to_s + '=', value)
+    #save(false)
+  end
```

Comment:
remove this method, i dont know why are you override a exist method in ActiveRecord? tell me why? it so danger